aspnetcore/blazor/javascript-interoperability/call-dotnet-from-javascript.md

```diff
@@ -1422,11 +1422,20 @@ Using JS modules to load JS is described in this article in the [JavaScript isol
 
 :::moniker-end
 
-<!-- UPDATE 8.0 Confirm for >=8.0 now that the warning is being removed. -->
+:::moniker range=">= aspnetcore-8.0"
+
+> [!WARNING]
+> Only place a `<script>` tag in a component file (`.razor`) if the component is guaranteed to adopt [static server-side rendering (static SSR)](xref:blazor/fundamentals/index#client-and-server-rendering-concepts) because the `<script>` tag can't be updated dynamically.
```

Comment:
@javiercn Is this warning worded strong enough for you? I remember you complaining about people trying to render `<script>` tags in triage.

Reply:
Seems fine to me. The reason for not using script tags in interactive components is clearly stated.